stock_putaway_product/models/product.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from openerp import models, fields
+
+_logger = logging.getLogger(__name__)
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    product_putaway_ids = fields.One2many(
+        comodel_name='stock.product.putaway.strategy',
+        related='product_variant_ids.product_putaway_ids',
+        string="Product stock locations")
+
```

Comment:
@JosDeGraeve After some tests (and intense reflection to understand :-)), it does not seem to work with variants...  With this related field, its impossible to create put-away strategy for two variants on same product.

Example: With demo data enable,  iPad Retina Display already has a put-away strategy for variant 16 GB, White, 2.4 GHz. If you try to create a strategy for another variant, it will raise a Constraint Error (on unique(putaway_id,product_product_id)).

The One2many widget try to add the new strategy on product.template.product_putaway_ids but with related, it seems that he write the new strategy on first variant (regardless of the product_product_id's value in putaway.strategy) .

I am a new developer Odoo, then a confirmation from an expert would be appreciated but I think you cannot write on this field...